src/lib/group/service.ts

```diff
@@ -143,6 +144,12 @@ export async function createGroupWithParticipants(
           deleteGroupParticipants({ groupParticipantId: gp.id }, t)
         )
       );
+      await Promise.all(
+        /* Get distinct group ids and disable each of them, if they end up empty */
+        [...new Set(userGroupParticipantsToDelete.map(gp => gp.groupId))].map(groupId =>
```

Comment:
Tenemos lodash ya metido en el proyecto, se puede hacer
Tenemos lodash ya metido en el proyecto, se puede hacer: 

```suggestion
       import { uniq } from 'lodash';
       // y aca 
        uniq(userGroupParticipantsToDelete.map(gp => gp.groupId)).map(groupId => disableGroupIfEmpty({ groupId })
```